### ISSUE TITLE
Hover: Images 404 and have bad alt text

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Demos-old#333]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/hover/

Many images are broken/404 on the interstitial page.

someone added "image" for alt text for every image :/

**Comment by alrra** [comment]
@dstorey @sarvaje

1. The code that seems to be on our side is not in this repository.
2. This demo is external, namely, it's from http://hover.ie/.

**Comment by molant** [comment]
@alrra we control the intestitial page so @dstorey 's issues should be fixed.

**Comment by alrra** [comment]
> we control the intestitial page

@molant Yes, but even that code is not in this repository.

**Comment by molant** [comment]
Yup, that's changed in the administration page. It is markdown so we might have an issue with some of the custom rules. I think you should be able to see it locally if you populate your db with the real flag.
